### DISCUSSION TITLE
How do I configure a theme to colorize json similar to jq?

### DISCUSSION THREAD

**Discussion by andresrinivasan** [sharkdp/bat#2391] [Q&A] [answered]
With `bat` I get

<img width="585" alt="Screen Shot 2022-11-01 at 9 54 36 AM" src="https://user-images.githubusercontent.com/134301/199291143-5684323e-87f9-40ca-bae8-f6c113739503.png">

I would really like something closer to `jq`

<img width="564" alt="Screen Shot 2022-11-01 at 9 55 21 AM" src="https://user-images.githubusercontent.com/134301/199291298-776b2584-acce-4b8c-b1cd-79d1a988a2f4.png">

I tried to play around with sublime themes but I don't understand how to link the moving parts together. Pointers in the right direction would be appreciated.

**Comment by sharkdp** [comment, marked as answer]
This depends on the theme. Unfortunately, most themes do not specify colors for the required scope. The default theme (Monokai Extended) should work though, because we [patched it](https://github.com/sharkdp/bat/blob/a7b0449967f640428eced369410622d53008f463/assets/patches/Monokai-Extended.tmTheme.patch#L7-L10). See also: #1099 #1400
